DFC.App.FindACourse/Controllers/CourseController.cs

```diff
@@ -119,20 +123,43 @@ public IActionResult Breadcrumb()
         }
 
         [HttpGet]
-        [Route("find-a-course/search/{articleName}/body")]
+        [Route("find-a-course/search/find-courses-to-get-a-job/body")]
+        [Route("find-a-course/search/find-courses-to-get-a-job")]
```

Comment:
@defc0nONE Hi Chris, just checking why do we need 2 different routes?

Reply:
This is the one I'm least unsure about (we were meant to have a discussion but it never came to be) and I've had a dig around the controller - this doesn't seem to be the first instance of this (for example, the `HttpGet` route at L77-80 also has similar route duplication and can be seen in a commit [here](https://github.com/SkillsFundingAgency/dfc-app-findacourse/pull/72/commits/54d6670012abdc096c50c7039c539e4dc5f48183)) and commits merely imply about getting pages 'working' so I wonder if this is a fundamental issue with how the application is built